libcaliptra/inc/caliptra_types.h

```diff
@@ -0,0 +1,52 @@
+// Licensed under the Apache-2.0 license
+#pragma once
+
+#include <stdint.h>
+
+typedef uint32_t caliptra_checksum;
+
```

Comment:
I think you are already aware of this, but just to document: I'm making this a moving target with https://github.com/chipsalliance/caliptra-sw/pull/534 that adds fips status to all the responses.

Reply:
Yep, I will be adjusting to account for it.